src/main/java/uk/ac/ebi/ena/util/SequenceAccessionUtil.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2020-2022 EMBL - European Bioinformatics Institute
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package uk.ac.ebi.ena.util;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.ac.ebi.embl.api.AccessionMatcher;
+import uk.ac.ebi.ena.exception.AccessionValidationException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class SequenceAccessionUtil {

```

Comment:
Utils does not tell anything much about functionality. Maybe call the class:
SequenceAccessionRangeSpliitter